nservicebus/recoverability/index.md

```diff
@@ -40,7 +40,7 @@ By default up to five immediate retries are performed if the message processing
 
 Note: The configured value describes the minimum number of times a message will be retried if its processing consistently fails. Especially in environments with competing consumers on the same queue, there is an increased chance of retrying a failing message more times across the endpoints.
 
-Note: Depending on the concurrency and transactionality settings of the endpoint immediate retries might happen even if configured to be off. To ensure that no retries is performed the endpoints needs to be configured to be either [non transactional](/transports/transactions.md) or the [concurrency restricted to 1](/nservicebus/operations/tuning.md).
+Note: Number of immediate retries is best effort. Immediate retries can be performed more than configured amount of time (even if configured to be off).
```

Comment:
Do we have specifics of the use cases/edge cases this might happen? Is this unique to one or more transports or is this a combination of several things? I don't know about this issue enough to provide anything else other than raising the questions customers would have after reading this section.

Reply:
It depends on the transport and the way it's configured. That said I'm not sure we can provide any valuable information in the context of a note. I think it's too little space.

I guess the decision we should make is if that change makes the note better or not. If not I propse to close this PR and handle the changes differently.

Reply:
This is information loss, as stated "To ensure no retries blabla set the to non-transactional"

You can also split it in multiple notes.

Reply:
@ramonsmits I've reintroduced note on the transaction mode. 

I also removed concurrency limit as this does not prevent from multiple processings in competing consumer setup.